@kiva/kv-stream/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@kiva/kv-stream",
+  "version": "1.0.0",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsup src/*.ts --format cjs,esm --dts --clean",
+    "lint": "eslint --ext .ts,.vue ./src",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kiva/kv-ui-elements.git"
+  },
+  "bugs": {
+    "url": "https://github.com/kiva/kv-ui-elements/issues"
+  },
+  "homepage": "https://github.com/kiva/kv-ui-elements#readme",
```

Comment:
Are these the correct urls for this project?

Reply:
This should be fine, since it's a shared repo.